### DISCUSSION TITLE
[projects] Priority levels. "Medium" should be "Normal"

### DISCUSSION THREAD

**Discussion by luispuerto** [community/community#36539] [Other Feature Feedback, Questions, & Ideas]
Hey! 

I really don't know if I should post here, but if there is a more suitable place, please tell me. 

I just started to use GitHub projects and I think they're really cool. 

However, I think that regarding the task priority categorization the `medium` tag should be named `normal` or perhaps something else. 

Why? mainly because I keep track of task in other system and I also use tags for that. There is already a medium tag for size, so there is no reason to have a medium tag for priority. Also, I think normal priority describes better priority than medium. 

How they are: 
<img width="259" alt="image" src="https://user-images.githubusercontent.com/22882880/196404522-3a5c326c-f92a-4d50-bb7d-e01e84ac3cdc.png">

How they should be (in my opinion)
<img width="232" alt="image" src="https://user-images.githubusercontent.com/22882880/196404484-1666b06e-bae8-42a3-8b34-cf5f63ef2878.png">

**Comment by ibrahimmemonn** [comment]
Look at the top you'll see "New Discussion" tap there and post whatever you want

**Comment by luispuerto** [comment]
@ibrahimmemonn ummm I mentioned that I didn't know if this is the correct place because there is no section for project feedback…

**Comment by ossanna16** [comment]
Hey there @luispuerto and thank you for asking a good question! I'm glad you found our community 🙂
